### PR TITLE
Laptop request design

### PR DESCRIPTION
**Pull request by Fareedh-12**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description
Laptop request and donator form design 

Write a summary of your change. Explain why you have made it.
created consistency with the other pages on the laptop request and laptop donation forms . 

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [x] I have commented my code
- [x] I have updated any documentation

### CODE REVIEW

**Comment by Fareedh-12** [comment]
@shieldo @gregdyke @ajbader-d @theseafaringturtle 
Please review my PR.

File: client/src/pages/component/BackgroundImageComponent/BackgroundImage.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.background-image-container {
+	background-image: url(../image/home-body.jpg) ;
+	background-repeat: no-repeat;
+    background-position: center center;
+	width: 100%;
+	height: 480px;
+	background-size: auto;
```
Comment (Abdalrazaq94) [inline client/src/pages/component/BackgroundImageComponent/BackgroundImage.css:7]: I think the background size should be contained so it works on all devices and contains the picture

File: client/src/pages/component/BackgroundImageComponent/BackgroundImage.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.background-image-container {
+	background-image: url(../image/home-body.jpg) ;
+	background-repeat: no-repeat;
+    background-position: center center;
```
Comment (Abdalrazaq94) [inline client/src/pages/component/BackgroundImageComponent/BackgroundImage.css:4]: background-position: should be cover

File: client/src/pages/component/BackgroundImageComponent/BackgroundImage.css (hunk 1 above)
Comment (Fareedh-12) [reply client/src/pages/component/BackgroundImageComponent/BackgroundImage.css:7]: Left it as auto

Comment (Abdalrazaq94) [reply client/src/pages/component/BackgroundImageComponent/BackgroundImage.css:7]: cool

**Pull request merged**